src/main/resources/Macros/MapMacro/MapMacro.xml

```diff
@@ -115,23 +115,23 @@ No content.
 == Map with parameters ==
 
 {{code}}
-{{map location="Rue des Thermopyles, 75014 Paris" width="400px" height="300px" library="googlemaps"}}
+{{map location="Rue des Thermopyles, 75014 Paris" width="500" height="400" library="googlemaps"}}
 A beautiful street in Paris
 {{/map}}
 {{/code}}
 
-{{map location="Rue des Thermopyles, 75014 Paris" width="400px" height="300px" library="googlemaps"}}
+{{map location="Rue des Thermopyles, 75014 Paris" width="500" height="400" library="googlemaps"}}
 A beautiful street in Paris
```

Comment:
I guess all the previous changes in this file are documentation improvements, right?

Reply:
These changes consist of documentation improvements indeed. I integrated the changes brought to  https://extensions.xwiki.org/xwiki/bin/view/Extension/Map%20Macro